src/features/wav_iterator.py

```diff
@@ -12,6 +12,7 @@
 from scipy.io import wavfile
 
 from . import spectral_features
+from . import mixer 
```

Comment:
From the way things are structured, it doesn't appear that `wav_iterator` should be imported into mixer. We're getting batches from mixer and not the other way around, i.e., we're not mixing anything from batcher...though that may be simpler and faster. Indeed, I mixer never calls `batcher`, the only import from `wav_iterator` into `mixer.py`.